spec/controllers/reports_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe ReportsController, type: :controller do
+  let!(:user){ create :user }
+  before :each do
+    sign_in user
+  end
+
+  describe 'GET #index' do
+    let!(:first_report){ FactoryGirl.create(:report, user: user)}
+    let!(:second_report){ FactoryGirl.create(:report, user: user)} 
+    
+    
+    it 'populate an array of all reports @reports' do
+      get :index
```

Comment:
I think it's better to move get :index to before(:each) block and call it before each test in this suite.